source/operations/concepts/scanner.rst

```diff
@@ -0,0 +1,102 @@
+.. _minio-concepts-scanner:
+
+==============
+Object Scanner
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO uses the built-in scanner to check objects for healing and to take any scheduled object actions.
+Such actions may include:
+
+- calculate data usage on drives
+- evaluate and apply configured :ref:`lifecycle management <minio-lifecycle-management>` or :ref:`object retention <minio-object-retention>` rules
+- perform :ref:`bucket <minio-bucket-replication>` or :ref:`site <minio-site-replication-overview>` replication
+- check objects for missing or corrupted data or parity shards and perform :ref:`healing <minio-concepts-healing>`
+
+The scanner performs these functions at two levels: cluster and bucket.
+At the cluster level, the scanner splits all buckets into groups and scans one group of buckets at a time.
+The scanner starts with new buckets, then randomizes the scanning of other buckets.
+The scanner completes checks on all bucket groups before starting over with a new set of scans.
+
+At the bucket level, the scanner groups items in buckets and scans selected items from that bucket.
+The scanner selects objects for a scan based on a hash of the object name.
+Over a span of 16 scans, MinIO checks every object in the namespace.
+
+Scan Length
+-----------
+
+Multiple factors impact the time it takes for a scan to complete.
+
+Some of these factors include:
+- Type of drives provided to MinIO
+- Throughput and :abbr:`iops <input/output operations per second>` available
+- Number and size of objects
+- Other activity on the MinIO Server
+
+For example, by default, MinIO pauses the scanner to make I/O operations available for read and write requests.
+This can lengthen the time it takes for a scan to complete.
+
+MinIO waits between each scan by a factor multiplication of the time it takes each scan operation to complete.
+By default, the value of this factor is ``10.0``, meaning MinIO waits 10x the length of an operation after one scan completes before starting the next scan.
+MinIO also applies a maximum wait time between operations, set to ``15s`` by default.
```

Comment:
Remove this. We use the `speed=slowest/slow/default/fast/fastest` which contains these values as presets.